cmd/soroban-cli/src/get_spec.rs

```diff
@@ -0,0 +1,69 @@
+use soroban_env_host::xdr;
+
+use soroban_env_host::xdr::{
+    ContractDataEntry, ContractExecutable, ScSpecEntry, ScVal, ScContractInstance,
+};
+
+use soroban_spec::read::FromWasmError;
+pub use soroban_spec_tools::contract as contract_spec;
+
+use crate::rpc;
+use crate::commands::{config::data, global};
+
+
+#[derive(thiserror::Error, Debug)]
+pub enum GetSpecError {
```

Comment:
This should just be `Error`, since we can refer to it externally with the module `get_spec::Error`.

Reply:
fixed